docs/source/configuration/advanced_configuration.md

```diff
@@ -238,7 +238,7 @@ data:
 Since both of the file names (`parameters.yml` and `parameters_globals.yml`) match the config pattern for parameters, the `OmegaConfigLoader` will load the files and resolve the placeholders correctly.
 
 #### Catalog
-From Kedro `0.18.10` templating also works for catalog files. To enable templating in the catalog you need to ensure that the template values are within the catalog files or the name of the file that contains the template values follows the same config pattern specified for catalogs.
+From Kedro `0.18.11` templating also works for catalog files. To enable templating in the catalog you need to ensure that the template values are within the catalog files or the name of the file that contains the template values follows the same config pattern specified for catalogs.
```

Comment:
I'm confused, tamplating in catalog worked from the previous release, right? Maybe a "search and replace" gone rogue, or am I missing something?

Reply:
It does still work, but as part of the release process we update all named instances of package versions in the docs

Reply:
But if it works from version 0.18.10 we should retain this one rather than update it, since it hasn't stopped working and we need to 'pin' the earliest version from which it works within the docs. Suggest this is one to roll back.